docs/website/developers/committers/release-process/make-release-artifacts.md

```diff
@@ -24,6 +24,8 @@ To run the script:
 It will show you the release information it has deduced, and ask yes-or-no if it can proceed. Please note that the
 script will thoroughly clean the Git workspace of all uncommited and unadded files.
 
+**You really probably want to run this against a secondary checkout.** It will wipe `.project` files and other IDE metadata, and bad things can happen if an IDE tries to write while the script is running. Also as it takes a long time, this means your workspace is not tied up. One quick and easy way to do this is to `git clone` the local directory of your primary checkout to a secondary location.
+
```

Comment:
(Minor) There's also a Vagrantfile for setting up a completely clean build environment in a VM. Maybe I'll add docs for that separately, however.